doc/docs-revamp/dev-onboarding-quick-setup.md

```diff
@@ -0,0 +1,375 @@
+---
+title: "Developer onboarding and quick setup guide"
+description: "setting up and testing developer environment, writing first smart contract"
+date: 2024-03-26
+---
+
+# Section 3. Developer onboarding and quick setup guide
+
+This guide's objective is to help you set up your development environment, test it, and write your first smart contract within an hour. This will be a simple vesting contract in which someone sends a gift of ada to a beneficiary. The beneficiary can access their gift only after a specific deadline has passed and the beneficiary has signed the contract. 
+
+## Setting up and testing your development environment
+
+### Prerequisites
+
+#### Hardware and OS requirements
+
+**Processing power**
+- Minimum: Intel Core i5 or AMD equivalent
```

Comment:
How were these requirements determined? I doubt you can't develop Plutus on an intel core i3, for example.

Reply:
I just did a little Googling to get a sense of some common recommended specs.